ygen/codegen.go

```diff
@@ -667,10 +670,10 @@ type mappedYANGDefinitions struct {
 //	- cfg: the current generator's configuration.
 // It returns a mappedYANGDefinitions struct populated with the directory and enum
 // entries in the input schemas, along with the calculated schema tree.
-func mappedDefinitions(yangFiles, includePaths []string, cfg GeneratorConfig) (mappedYANGDefinitions, []error) {
+func mappedDefinitions(yangFiles, includePaths []string, cfg GeneratorConfig) (*mappedYANGDefinitions, []error) {
```

Comment:
If it's not too painful of a change, consider passing cfg by ptr too. Not for performance but rather to stick with convention. Sorry, should have mentioned this earlier.

Reply:
Good catch, done.